.github/workflows/python_ci.yml

```diff
@@ -23,6 +23,11 @@ on:
         description: "Flink version to test against."
         required: true
         type: string
+      jdk_version:
+        description: "Jdk version to test against."
+        required: false
+        default: 8, 11
```

Comment:
Does this add value to default to 2 different versions?
The python test infra creation takes serious time, and I am not sure that running it twice worth the additional resources committed.